pyspark_ai/tool.py

```diff
@@ -104,3 +104,104 @@ def _run(
 
     async def _arun(self, *args: Any, **kwargs: Any) -> Any:
         raise NotImplementedError("ListTablesSqlDbTool does not support async")
+
+
+class ColumnQueryTool(BaseTool):
```

Comment:
I would suggest let's keep it simpler and add tool SimilarValueTool only in this PR